src/app/layouts/button/button.component.ts

```diff
@@ -7,6 +7,7 @@ import { Component, Input, Output, EventEmitter } from '@angular/core';
 })
 export class ButtonComponent {
     @Input() type: string;
+    @Input() disabled: boolean;
```

Comment:
Should this property have a default value (false)? For buttons on master view, for example.